tests/testsuite/login.rs

```diff
@@ -66,83 +59,6 @@ fn check_token(expected_token: &str, registry: Option<&str>) -> bool {
     }
 }
 
-#[cargo_test]
-fn login_with_old_credentials() {
-    registry::init();
-
-    cargo_process("login --host")
-        .arg(registry_url().to_string())
-        .arg(TOKEN)
-        .run();
-
-    // Ensure that we get the new token for the registry
-    assert!(check_token(TOKEN, None));
```

Comment:
I believe these tests should largely all get updated to `--index` unless they're explicitly testing that `--host` produces a deprecated error message.

Reply:
Thank you for your reply. Actually, ```--registry``` is used to replace ```--host``` in here. The test case of cargo login ```--registry``` is already available below. There seems to be no need for modification here.

Reply:
If `--registry` is the replacement for` --host` (I forget what all these flags do) then we can't delete the warning because the warning says "Please use '--index' instead.". If `--index` works then these tests should use `--index`.

Reply:
Thanks for replying, alex. As I understand it, in the ```cargo login``` subcommand, use ```--registry``` instead of ```--host```; In the ```cargo publish``` and ```cargo search``` subcommand, use ```--index``` instead of ```--host```.
The ```--host``` command is now redundant, and both the ```cargo login --registry``` and ```cargo publish --index``` tests exist.